### PR TITLE
[W5.11][W10-A2]Paul-Henri MIGNOT

### PR DESCRIPTION
**Pull request by phmignot**
Adding the Edit command, just able to edit name, but others contact information will be easy to add to this command.

### CODE REVIEW

File: src/seedu/addressbook/commands/EditCommand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/EditCommand.java]: Don't import everything with *, import only specific classes that are necessary

File: src/seedu/addressbook/commands/EditCommand.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            addressBook.editPerson(oldProfile, newProfile);
+            return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, newProfile));
+        } catch (IndexOutOfBoundsException ie) {
+            return new CommandResult(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        } catch (UniquePersonList.DuplicatePersonException dpe) {
+            return new CommandResult(MESSAGE_DUPLICATE_PERSON);
+        }
+    }
+    
+    private void InitiateProfiles() throws IndexOutOfBoundsException {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/EditCommand.java]: Please add header comments for non-trivial methods

**Review by okkhoy — COMMENTED** [review]
Good to see you have updated the JUnit tests as well :+1:

**Comment by okkhoy** [comment]
@phmignot 
- In general, good work on this. Keep it up!
Some comments added. Please ack & close the PR after reading comments.

**Comment by phmignot** [comment]
Changes applied

**Pull request closed without merging**